docker-compose.override.yml

```diff
@@ -13,6 +13,12 @@ services:
     environment:
       - POSTGRES_HOST=postgres_docker
       - REDIS_HOST=redis
+      - SMTP_HOST=mailcatcher
```

Comment:
Can we keep these variables in `.env`? I generally run the dbs/services via docker, but the actual app locally on the host machine.

Reply:
Of course, I can always put them in .env myself, but I guess the question is - what's the advantage in any case to putting them here vs. `.env.example`?

Reply:
They will work just as well locally. I can add them to the .env.example file. The only advantage to having them in that file is docker can resolve the dns so you don't have to have your own ip address there (localhost doesn't work because docker). I would prefer to keep them in the docker-compose, but I will add documentation about how to run locally too. Does that work for you?

Reply:
Good point about localhost - how about we move all but `SMTP_HOST` to `.env.example` and then keep host here and commented out in `.env.example`. Might be a decent pattern to follow for minio & clamav as well.